js_modules/dagit/packages/core/src/schedules/SchedulesCheckAll.tsx

```diff
@@ -0,0 +1,24 @@
+import {Checkbox, Tooltip} from '@dagster-io/ui';
+import * as React from 'react';
+
+interface Props {
+  checkedCount: number;
+  totalCount: number;
+  onToggleAll: (checked: boolean) => void;
+}
+
+export const SchedulesCheckAll = ({checkedCount, totalCount, onToggleAll}: Props) => {
```

Comment:
It doesn't look like there's anything schedule specific about this -- maybe if we give it a generic name it'll have a better shot at getting re-used?

Reply:
Yep, this will be renamed `CheckAllBox` in a followup for Sensors, later today.